packages/expo-updates/ios/EXUpdates/Logging/UpdatesLogEntry.swift

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+// Schema for the fields in expo-updates log message JSON strings
+
+import Foundation
+
+public struct UpdatesLogEntry: Codable {
```

Comment:
is it possible to expose this to obj-c if it also subclasses NSObject? Seems like it would be nice if the class calling `getLogEntries` weren't also responsible for parsing the entries, but also understandable if it's too tricky to implement across both languages.

Reply:
Sure I can have a look

Reply:
Exposing this to Objective C would be a complete reimplementation and be a bit awkward. It might be better to change the reader to return a list of `NSDictionary` objects for Objective-C usage.

Reply:
Cool -- that makes sense to me. Thanks for checking that out.

Reply:
Now Objective-C can call the reader and get entries returned as NSDictionaries.